src/volume_mgr.rs

```diff
@@ -993,30 +1015,30 @@ where
     }
 
     fn get_volume_by_id(&self, volume: RawVolume) -> Result<usize, Error<D::Error>> {
-        for (idx, v) in self.open_volumes.iter().enumerate() {
-            if v.volume_id == volume {
-                return Ok(idx);
-            }
-        }
-        Err(Error::BadHandle)
+        self.open_volumes
```

Comment:
I’m not sure this is a big enough improvement to be worth the change?

Reply:
Yeah not really an improvement per se, the assembly is nearly identical between the two. More a style preference, on for-loops vs iterators. No problem reverting this

Reply:
I think that would make sense